README.md

```diff
@@ -56,51 +51,88 @@ function Worker() {
+
+Upon construction of the worker, the worker options `allowOrigins` property is read into the construction of the `OriginAccessManager`. Properties of `allowOrigins` translate to a _purpose_ on the OAM, with their values, being a list of origins, are added under that _purpose_ (and `null` _key_). The `any` property translates to a `null` _purpose_, which matches any _purpose_. For example, `isAllowed` will return `true` for origins stored under a `null` _purpose_, regardless the _purpose_ and _key_ combination queried.
+
+### OriginAccessManager Methods
+- `add(origin, purpose, key):` Adds (allows) the _origin_ under the _purpose_ and _key_. A `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+- `remove(origin, purpose, key):` Removes (un-allows) the _origin_ under the _purpose_ and _key_. A `null` purpose and/or key will __not__ match any _purpose_ and/or _key_.
+- `remove_byKey(key):` Removes all origins for all purposes under the _key_. A `null` _key_ is not accepted, must be a string.
+- `getAllowList(purpose, key):` returns a list of origins under the _purpose_ and _key_. A `null` _purpose_ is not accepted, must be a string. Previously added origins under `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+- `isAllowed(origin, purpose, key):` returns `true` if _origin_ is allowed under the _purpose_ and _key_. A `null` _purpose_ is not accepted, must be a string. Previously added origins under `null` _purpose_ and/or _key_ will match any _purpose_ and/or _key_, respectively.
+# Change log
+- __1.0.0__ - Feb 28, 2023
+  - Inital Release
+- __1.0.1__ - Mar 1, 2023
+  - Added src/ directory to published files to support source map
+- __1.0.2__ - Mar 23, 2023
+  - local storage will only save paymentAddress and maxWorkingSandbox props
+  - workerOptions source always coming from dcpConfig
+  - added delay between quick worker fetching states
+  - Quality of life + maintainability improvements
+- __2.0.0__ - May
+  - now returns the worker
+  - proper handling of race-condition in constructing the worker when/if the hook is executed multiple times at once
+  - workerOptions passed to worker constructor is a Proxy now
+    - needed reflect changes to paymentAddress and maxWorkingSandbox props to local storage and trigger component update/re-render
+  - removed workerState.isLoaded since now we are returning the worker
+    - !isLoaded === !worker
+  - removed workerOptionsState, start/stop/toggleWorker, sandboxes, originManager
+    - these are all accessible via Worker API
+  - removed applyWorkerOptions since editing workerOptions should be done by directly mutating worker.workerOptions
+  - improved error handling
+  - quality of life improvements
```

Comment:
**suggestion:** Order the Changelog in reverse chronological order.

Most folks want to see the latest changes described first [^1]. e.g., On a new release, they'll want to see the new changes listed first in the file, not at the end of it after scrolling down.

[^1]: https://keepachangelog.com/en/1.0.0/#how